<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>connectDb</class>
 <widget class="QMainWindow" name="connectDb">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>722</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Berlin Sans FB</family>
    <pointsize>8</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>herbsWiki</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/imgs/img/icon.png</normaloff>:/imgs/img/icon.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>50</width>
    <height>50</height>
   </size>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <family>Berlin Sans FB</family>
       </font>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(247, 246, 255);</string>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideRight</enum>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/imgs/img/home.png</normaloff>:/imgs/img/home.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Home</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>100</y>
          <width>431</width>
          <height>301</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Myanmar Text</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-color: rgb(85, 170, 0);
background-color: rgb(101, 170, 120);</string>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QLineEdit" name="lineEditSname">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>30</y>
           <width>261</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Berlin Sans FB</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(106, 106, 106);
background-color: rgb(255, 255, 255);
border: 1px solid white;
border-radius: 20px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>  Scientific name :</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEditCname">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>90</y>
           <width>261</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Berlin Sans FB</family>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(106, 106, 106);
background-color: rgb(255, 255, 255);
border: 1px solid white;
border-radius: 20px;</string>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string notr="true">  Common name :</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEditFamily">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>150</y>
           <width>261</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Berlin Sans FB</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(106, 106, 106);
background-color: rgb(255, 255, 255);
border: 1px solid white;
border-radius: 20px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>  Family :</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEditUse">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>210</y>
           <width>261</width>
           <height>61</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Berlin Sans FB</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(106, 106, 106);
background-color: rgb(255, 255, 255);
border: 1px solid white;
border-radius: 27px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string> Use :</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>410</y>
          <width>431</width>
          <height>80</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>4</pointsize>
         </font>
        </property>
        <property name="title">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>7</y>
           <width>411</width>
           <height>61</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="pButtonAddherb">
            <property name="font">
             <font>
              <family>Berlin Sans FB</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 85, 127);
border: 8px solid rgb(0, 85, 127);
border-radius: 4px;</string>
            </property>
            <property name="text">
             <string>Add herb</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pButtonUpdate">
            <property name="font">
             <font>
              <family>Berlin Sans FB</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 85, 127);
border: 8px solid rgb(0, 85, 127);
border-radius: 4px;</string>
            </property>
            <property name="text">
             <string>Update</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pButtonDelete">
            <property name="font">
             <font>
              <family>Berlin Sans FB</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(204, 6, 26);
color: rgb(255, 255, 255);

border: 8px solid rgb(204, 6, 26);
border-radius: 4px;</string>
            </property>
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="widget" native="true">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>0</y>
          <width>301</width>
          <height>101</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/imgs/img/logo2.png);</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/imgs/img/search.png</normaloff>:/imgs/img/search.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Search</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>471</width>
          <height>61</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>7</y>
           <width>411</width>
           <height>51</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="lineEditSearch">
            <property name="styleSheet">
             <string notr="true">color: rgb(106, 106, 106);
background-color: rgb(255, 255, 255);
border: 1px solid white;
border-radius: 10px;</string>
            </property>
            <property name="placeholderText">
             <string> Scientific name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pButtonSearch">
            <property name="font">
             <font>
              <family>Berlin Sans FB</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="statusTip">
             <string/>
            </property>
            <property name="whatsThis">
             <string/>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 85, 127);
border: 8px solid rgb(0, 85, 127);
border-radius: 4px;</string>
            </property>
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QTreeView" name="treeViewSearch">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>471</width>
          <height>192</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Berlin Sans FB</family>
         </font>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="autoScroll">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/imgs/img/leaf.png</normaloff>:/imgs/img/leaf.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Wiki</string>
       </attribute>
       <widget class="QTreeView" name="treeView">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>491</width>
          <height>491</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Berlin Sans FB</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(130, 130, 130);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="midLineWidth">
         <number>0</number>
        </property>
        <property name="autoScroll">
         <bool>false</bool>
        </property>
        <property name="allColumnsShowFocus">
         <bool>true</bool>
        </property>
        <attribute name="headerHighlightSections">
         <bool>true</bool>
        </attribute>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>20</y>
          <width>131</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Berlin Sans FB</family>
         </font>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;span style=&quot; font-size:6pt;&quot;&gt;Click here to search herbs through web&lt;/span&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 85, 127);
border: 8px solid rgb(0, 85, 127);
border-radius: 4px;</string>
        </property>
        <property name="text">
         <string>Web search</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/imgs/img/wwww.png</normaloff>:/imgs/img/wwww.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>540</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
